tests/unit/fixtures/gameStates/points.js

```diff
@@ -137,10 +137,10 @@ export const points = {
             lockedAt: null,
             match: null,
             name: 'Test Game',
-            p0: gameFixture.p0,
+            p0: gameFixture.p0.id,
```

Comment:
updated the way matches come throught to the client so `p0` and `p1` on all the individual games are just the player ids, not the objects